docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -595,3 +596,27 @@ df.select { (colsOf<Int>() and age).distinct() }
 
 <inline-frame src="resources/org.jetbrains.kotlinx.dataframe.samples.api.Access.columnSelectorsModifySet.html" width="100%"/>
 <!---END-->
+
+### Column Resolvers
+
+`ColumnResolver` is the base type used to access columns within the **Columns Selection DSL**,  
+as well as the return type of columns selection expressions.
+
+All functions described above for selecting columns in various ways return a `ColumnResolver` of a specific kind:
+
+- **`SingleColumn`** — resolves to a single [`DataColumn`](DataColumn.md).
+- **`ColumnAccessor`** — a specialized `SingleColumn` with a defined path and type argument.  
+  It can also be renamed during selection.
+  - **`ColumnPath`** — a wrapper for [`DataColumn`](DataColumn.md) path
+    in [`DataFrame`](DataFrame.md) also can serve as a `ColumnAccessor`.
+```kotlin
+// Select all columns from the group by path "group2"/"info":
+df.select { pathOf("group2", "info").cols() }
```

Comment:
That's `.allCols()`, isn't it?
Using Korro here would catch that.

Reply:
`allCols()` has compiler plugin support :) `cols {}` is a filter. (we should probably remove the default `{ true }` predicate in cols, come to think of it.

Reply:
okay not yet for column paths, but it's generally good advice when you don't want to filter